gems/aws-sdk-core/lib/seahorse/client/h2/connection.rb

```diff
@@ -0,0 +1,196 @@
+if RUBY_VERSION >= '2.3'
+  require 'http/2'
+end
+require 'openssl'
+require 'socket'
+
+module Seahorse
+  module Client
+    # @api private
+    module H2
+
+      # H2 Connection build on top of `http/2` gem
+      # with TLS layer plus ALPN, requires:
+      # Ruby >= 2.3 and OpenSSL >= 1.0.2
+      class Connection
+
+        OPTIONS = {
```

Comment:
How are we surfacing these defaults to user-facing documentation?

Reply:
Addressed in the plugin file doc_strings :)